README.md

```diff
@@ -80,6 +81,17 @@ See below for how to configure the environment variables appropriately
 - Barry__ApiKey=XXXXX # See below Barry Access Token section
 - Barry__MPID=YYYYY # See below Barry MPID section
 ```
+### Energinet
+
+```yaml
+- TeslaMate__EnergyProvider=Energinet
+- Energinet__Region=YYYYY # See below a list of all Energinet regions
+- Energinet__VAT=1.25 # VAT multiplier. In this example 25%
+- Energinet__FixedPrice__TimeZone=Europe/Copenhagen # IANA (tz database) time zone code, used for below times 
```

Comment:
You are missing an 's' here
Energinet__FixedPrices_TimeZone works.

Reply:
Thanks! I have fixed the typo